include/lomse_im_note.h

```diff
@@ -134,9 +134,9 @@ class ImoNoteRest : public ImoStaffObj
     ImoTuplet* get_first_tuplet();
 
     //IM attributes interface
-    virtual void set_int_attribute(TIntAttribute attrib, int value);
-    virtual int get_int_attribute(TIntAttribute attrib);
-    virtual list<TIntAttribute> get_supported_attributes();
+    virtual void set_int_attribute(TIntAttribute attrib, int value) override;
```

Comment:
Thanks for this PR. 
The changes in this file create a conflict. Probably your fork is not updated to latest code. The 'override' atrribute was fixed in all lomse code  (I hope) in a previous PR and, for sure, it is already fixed in this file. Please remove this file from the PR. Thank you

Reply:
Reverted (I hope!)